app/helpers/book_helper.rb

```diff
@@ -0,0 +1,8 @@
+module BookHelper
+  def delete_book_link(book)
+    link_to t('books.delete.link'),
+    book,
+    method: :delete,
+    data: { confirm: t('books.delete.confirmation_text') }
```

Comment:
Line 4-6 should be indented by two spaces. BTW I think you should try adding Hound so it can catch some of this early on. It's really handy!